libs/newsletter-workflow/src/lib/steps/newsletterData/createDraftNewsletterLayout.ts

```diff
@@ -31,4 +32,6 @@ The first step is to enter the name of your newsletter, for example **Down to Ea
 	},
 	schema: formSchemas.startDraftNewsletter,
 	role: 'CREATE_START',
+	canSkipTo: true,
+	executeSkip: executeModify,
```

Comment:
Skipping isn't working here - the "invalid or missing listId" message comes up.  This is because the "executeCreate" function (the 'next' button's `executeStep` ) needs to be be called to add the listId to the draftData before the data can be saved.

I think we could fix this by using "executeCreate" for executeSkip too - it would still require the user to enter a name before skipping. We need it to do this because "executeCreate" also generates the derived fields from the name currently. 

I think on a future PR, we should move the derived fields logic to the launch wizard - "executeCreate" would still set the "listId" (if not already set), but we wouldn't need to have the name set, so users could freely skip from this step.

Also, if we defer the derived fields until launch, they will be based on the name as it is at that point - currently, if you changed the name after setting it, the derived fields would still be based on the original name.

Reply:
Well spotted - thanks. Sounds sensible.